vtysh/vtysh.c

```diff
@@ -680,7 +680,7 @@ int vtysh_mark_file(const char *filename)
 	while (fgets(vty->buf, VTY_BUFSIZ, confp)) {
 		lineno++;
 		tried = 0;
-		strcpy(vty_buf_copy, vty->buf);
+		strlcpy(vty_buf_copy, vty->buf, VTY_BUFSIZ);
```

Comment:
can you convert this (and the `fgets` above) to `sizeof()` while at it?

(and maybe other occurrences of it too...)

Reply:
No? `vty->buf` is a `char *`

Reply:
you're right, _sigh_